app/controllers/events/view/tickets/attendees/list.js

```diff
@@ -74,7 +74,7 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
     const attendee = this.store.peekRecord('attendee', attendee_id, { backgroundReload: false });
     attendee.toggleProperty('isCheckedIn');
     if (attendee.isCheckedIn) {
-      const newCheckinTimes = attendee.get('checkinTimes') === null ? `${moment().toISOString()}` : `${attendee.get('checkinTimes')},${moment().toISOString()}`;
+      const newCheckinTimes = attendee.get('checkinTimes') === null ? `${moment().toISOString()}` : `${moment().toISOString()}`;
```

Comment:
![20](https://user-images.githubusercontent.com/72552281/121330611-caed5a80-c933-11eb-9f55-93bebb92d1e5.png)

Server is already merging times.